pkg/graph/resolvers/operational_solution_and_model_plan.go

```diff
@@ -14,3 +14,18 @@ func ModelPlansByOperationalSolutionKey(
 ) ([]*models.ModelPlanAndOperationalSolution, error) {
 	return store.ModelPlanGetByOperationalSolutionKey(logger, operationalSolutionKey)
 }
+
+// ModelPlanByOperationalSolutionStatusPlannedActiveOrEnded converts a Model status to a ModelBySolutionStatus
+func ModelPlanByOperationalSolutionStatusPlannedActiveOrEnded(modelPlanStatus models.ModelStatus) models.ModelBySolutionStatus {
```

Comment:
Can you rename this function to match the new name in the GQL schema?

Reply:
Good call! Thanks!

Reply:
[23ee4d0](https://github.com/CMSgov/mint-app/pull/1233/commits/23ee4d08764826ad992fe8ae9f297dd668bd48a5)